src/RapidezStatamicServiceProvider.php

```diff
@@ -2,85 +2,85 @@
 
 namespace Rapidez\Statamic;
 
-use Illuminate\Support\ServiceProvider;
-use Rapidez\Statamic\Http\Controllers\StatamicRewriteController;
-use Rapidez\Statamic\Listeners\EntrySavedListener;
-use Statamic\Events\EntrySaved;
 use Illuminate\Support\Facades\Cache;
 use Illuminate\Support\Facades\Event;
-use Rapidez\Statamic\Commands\DeleteProductsCommand;
-use Rapidez\Statamic\Commands\SyncProductsCommand;
-use Rapidez\Statamic\Commands\SyncCategoriesCommand;
-use Statamic\Facades\Entry;
-use Rapidez\Statamic\Http\ViewComposers\StatamicDataComposer;
-use Statamic\Statamic;
 use Illuminate\Support\Facades\View;
+use Illuminate\Support\ServiceProvider;
 use Illuminate\View\View as RenderedView;
-use Statamic\Facades\Site;
 use Rapidez\Core\Facades\Rapidez;
-use Validator;
-use Statamic\Events\GlobalSetSaved;
+use Rapidez\Statamic\Http\Controllers\StatamicRewriteController;
+use Rapidez\Statamic\Http\ViewComposers\StatamicGlobalDataComposer;
 use Statamic\Events\GlobalSetDeleted;
+use Statamic\Events\GlobalSetSaved;
+use Statamic\Facades\Entry;
+use Statamic\Facades\Site;
+use Statamic\Statamic;
 
 class RapidezStatamicServiceProvider extends ServiceProvider
 {
     public function boot()
     {
-        $this->app->singleton(StatamicDataComposer::class);
-
-        $this->bootCommands()
+        $this
             ->bootConfig()
+            ->bootViews()
             ->bootPublishables()
             ->bootComposers()
             ->bootListeners()
-            ->bootRoutes();
+            ->bootRoutes()
+            ->bootRunway();
```

Comment:
This is a small nitpick, but (in my opinion) if we're going to have any specific ordering of chaining here the functions below should at least be in the same order as they are called up here.

Reply:
Fixed